include/blogPost.php

```diff
@@ -1,62 +1,52 @@
 <?php
-function getAllBlogPosts(){
-    $post = array(
-        array(
-            'postId'=>0,
-            'title'=> "Getting To Know Me <br/>",
-            'body'=> "<div class = 'aboutMe'>
-                    Below are some more facts about me: <br/> <br/>
-                    I am from NY <br/>
-                     My favorite dessert is ice cream <br/>
-                     My least favorite forrd is chocolate <br/>
-                     I am scared of cats <br/>
-             My favorite Disney movie is mulan <br/><br/><br/>"
-         ),
-         array(
-             'postId'=>1,
-             'title'=> "What I am Studying <br/>",
-             'body'=> "<div class = 'aboutMe'>
-                     I am currently studying CogNeuro<br/><br/><br/>",
-          ),
-           array(
-               'postId'=>2,
-               'title'=> "What I'm Listening to <br/>",
-               'body'=> "<div class = 'aboutMe'>
-                    Explore the album covers to see what I am listening to!<br/><br/><br/>",
-              ));
-          return $post;
-}
-// include_once('config/init.php');
-// function getAllBlogPostsPost($postID){
-// $Result = dbQuery("
-//     SELECT *
-//     FROM blog_post
-//     WHERE blogPostId =:blogPostId
-//     ", array("blogPostId"=>$postId));
-//     return $Result;
-// }
-
-function echoPostLinkHtml(){
-    $post = getAllBlogPosts();
-    for ($i = 0; $i<count($post); $i++){
-        $postId = $post[$i]['postId'];
-        $postTitle = $post[$i]['title'];
-        $postBody= $post[$i]['body'];
-      echo  "<a class = 'resume' href = 'viewPost.php?postId=". $postId."'>".$postTitle." </a><br/>";
+include_once('config/init.php');
+function blogTagOrganizer (){
+    $tagTypes = dbQuery("
+            SELECT *
+            FROM tags
+            ")->fetchALL();
+    $tagArray = array();
+        foreach ($tagTypes as $tagsOrganized){
+            if(!in_array($tagsOrganized['tagName'], $tagArray)){
+                array_push($tagArray, $tagsOrganized['tagName']);
```

Comment:
Just so you know, there's a slightly shorter syntax for this. You can write
```
$tagArray[] = $tagsOrganized['TagName'];
```

The `$tagArray[]` just means "add to the end of the array". Your version is perfectly fine, so use whichever you prefer, I just wanted to make sure you know both options.